src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -300,7 +300,22 @@ export const QueryResultPane = () => {
         }
     });
 
-    return (
+    return !metadata || !hasResultsOrMessages(metadata) ? (
+        <div>
+            <div>{locConstants.queryResult.noResultMessage}</div>
```

Comment:
Based on the screenshot, this pane need some margins so that the text has some breathing room.

Reply:
@jdesai-24 for some design feedback here.

Reply:
Yeah the styling of this placeholder message definitely needs some improvement, I'll create an issue tracking this.

Reply:
https://github.com/microsoft/vscode-mssql/issues/18193